apps/marketplace/components/SellerAssessment/SellerAssessmentView.js

```diff
@@ -0,0 +1,105 @@
+      <p>
+        You cannot edit your request for approved categories. If you want to change your rate, please{' '}
+        <a
+          href="https://marketplace1.zendesk.com/hc/en-gb/requests/new"
+          rel="noopener noreferrer"
+          target="_blank"
+          className={styles.marginRight1}
+        >
+          contact our support team.
+        </a>
+      </p>
+    )}
+
+    {props.evidence.status === 'submitted' && (
+      <p>
+        You cannot edit your request once you&apos;ve submitted an request for assessment.
+        <br />
+        If you have an issue,{' '}
+        <a
+          href="https://marketplace1.zendesk.com/hc/en-gb/requests/new"
+          rel="noopener noreferrer"
+          target="_blank"
+          className={styles.marginRight1}
+        >
+          contact our support team.
+        </a>
+      </p>
+    )}
+
+    {props.evidence.status === 'rejected' && (
+      <p>
+        If your assessment is not approved, you can view the
+        <a href={`${rootPath}/seller-assessment/${props.evidence.id}/feedback/`}> feedback </a>
+        and submit another request.
+      </p>
+    )}
+
+    <AUheading level="2" size="lg">
+      Maximum daily rate
+    </AUheading>
+    <p>${props.evidence.maxDailyRate} (including GST)</p>
+    <AUheading level="2" size="lg">
+      Evidence
+    </AUheading>
+
+    {props.evidence.criteria &&
+      props.evidence.criteria.map(criteriaId => (
+        <React.Fragment key={criteriaId}>
+          <AUheading level="2" size="md">
+            Criteria
+          </AUheading>
+          <p className={styles.reviewText}>{props.evidence.domainCriteria[criteriaId].name}</p>
+          <AUheading level="2" size="md">
+            Client
+          </AUheading>
+          <p className={styles.reviewText}>{props.evidence.evidence[criteriaId].client}</p>
+          <AUheading level="2" size="md">
+            Referee&apos;s name and number
+          </AUheading>
+          <p className={styles.reviewText}>
+            {props.evidence.evidence[criteriaId].refereeName}: {props.evidence.evidence[criteriaId].refereeNumber}
+          </p>
+          <AUheading level="2" size="md">
+            Project date
+          </AUheading>
+          <p className={styles.reviewText}>
+            {props.evidence.evidence[criteriaId].startDate} - {props.evidence.evidence[criteriaId].endDate}
+          </p>
+          <AUheading level="2" size="md">
+            Background
+          </AUheading>
+          <p className={styles.reviewText}>{props.evidence.evidence[criteriaId].background}</p>
+          <AUheading level="2" size="md">
+            Evidence of meeting the criteria
+          </AUheading>
+          <p className={styles.reviewText}>{props.evidence.evidence[criteriaId].response}</p>
+          {props.evidence.criteria.indexOf(criteriaId) !== props.evidence.criteria.length - 1 && (
+            <div className={styles.spacer} />
+          )}
```

Comment:
Did you try passing [index as a parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) (eg. `props.evidence.criteria.map( (criteriaId, index) => ... )`?  You may not need to call `indexOf`.